src/Files.App/Data/Items/DriveItem.cs

```diff
@@ -214,12 +215,16 @@ public object ToolTip
 						Width = 16
 					}
 				};
-				itemDecorator.Click += ItemDecorator_ClickAsync;
+
+				ToolTipService.SetToolTip(itemDecorator, "SideBarEjectDevice/Text".GetLocalizedResource());
+
+				itemDecorator.Click += ItemDecorator_Click;
+
 				return itemDecorator;
 			}
 		}
 
-		private async void ItemDecorator_ClickAsync(object sender, RoutedEventArgs e)
+		private async void ItemDecorator_Click(object sender, RoutedEventArgs e)
```

Comment:
I think you should revert this as per #13533

Reply:
Let's keep as is. `Click` events are probably better like this.

Reply:
They did a bit confusing work. Practically, we should not append Async suffix to event methods.

Reply:
Ok then

Reply:
Sure 👍